code/modules/clothing/chameleon.dm

```diff
@@ -72,7 +72,8 @@
 	name = "Select Chameleon Outfit"
 	button_icon_state = "chameleon_outfit"
 	var/list/outfit_options //By default, this list is shared between all instances. It is not static because if it were, subtypes would not be able to have their own. If you ever want to edit it, copy it first.
-
+	syndicate = TRUE
```

Comment:
```suggestion
```
Remove this, the only thing that grants this action is the item specific action which is itself checking for syndi or not. Locking this to syndicate prevents the variant that is not syndicate from showing this button